scripts/component_updation/main.go

```diff
@@ -4,7 +4,7 @@ Uses a spreadsheet of centralized information about MeshModel components and the
 
 Usage: (order of flags matters)
 
-    ./main [--update_doc] [path-to-spreadsheet] [--only-published]
+    ./main [path-to-spreadsheet] [--update_doc] [relative path to docs in website] [--only-published]
```

Comment:
@Revolyssup will you take a moment to reflect on the use of this script as defined in this workflow? https://github.com/meshery/meshery/blob/2265b0064ec333726a68658d19dbdd6e2e39d327/.github/workflows/integrations-updater.yml#L42 

In the event that a maintainer invokes the workflow, and includes a an optional flag, the sequence of input arguments might be unexpected.